crates/acvm_backend_barretenberg/src/proof_system.rs

```diff
@@ -173,22 +166,6 @@ impl ProofSystemCompiler for Barretenberg {
         }
         .run())
     }
-
-    fn proof_as_fields(
-        &self,
-        _proof: &[u8],
-        _public_inputs: WitnessMap,
-    ) -> Result<Vec<FieldElement>, Self::Error> {
```

Comment:
We should keep these two methods in mind as we need them for recursion. Not sure if an issue should be made or if they should be left as is